src/main/java/com/google/jenkins/plugins/computeengine/ssh/SshKeysHelper.java

```diff
@@ -0,0 +1,162 @@
+package com.google.jenkins.plugins.computeengine.ssh;
+
+import static com.google.api.client.util.Throwables.propagate;
+import static com.google.common.base.Joiner.on;
+import static com.google.common.base.Splitter.fixedLength;
+import static org.jclouds.crypto.Pems.PRIVATE_PKCS1_MARKER;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.primitives.Bytes;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.math.BigInteger;
+import java.nio.charset.StandardCharsets;
+import java.security.KeyPair;
+import java.security.KeyPairGenerator;
+import java.security.NoSuchAlgorithmException;
+import java.security.PrivateKey;
+import java.security.SecureRandom;
+import java.security.interfaces.RSAPrivateCrtKey;
+import java.security.interfaces.RSAPrivateKey;
+import java.security.interfaces.RSAPublicKey;
+import java.util.Base64;
+import java.util.List;
+import java.util.Map;
+import org.jclouds.ssh.SshKeys;
+
+/** Methods copied from jclouds and modified to avoid missing dependencies in BaseEncoding */
+public class SshKeysHelper {
```

Comment:
This isn't an ideal solution to this problem, but I've isolated these additions in this class with one method (generate()) publicly available to use in GoogleKeyPair. This way, it can be easily replaced by a more permanent solution for generating SSH key pairs without jclouds.

Reply:
I would rather us not get into the business of generating our own key pairs. Suggest instead using another library: http://www.jcraft.com/jsch/examples/KeyGen.java.html

Reply:
Done